test/integration/supported/client-execute-tests.js

```diff
@@ -217,8 +221,9 @@ describe("Client @SERVER_API", function () {
             ];
             // no hint
             insertSelectTest(client, table, columns, values, null, done);
-        });
-        vit(
+        }); */
+        // No support for parameter hints
+        /* vit(
```

Comment:
I strongly reject such practice of commenting out test cases without having them documented clearly and marked with a tag for easy lookup.
Tests are here to reliably measure our progress; we mustn't decrease their accuracy.

Reply:
How would you suggest to mark those tests that are currently failing due to lack of nesesery features in the driver then?

Reply:
At least with a `TODO:`. But it's best to come up with a dedicated keyword, e.g. `TODO: TESTING:` or something like this.